client/js/jsonedit.js

```diff
@@ -668,6 +677,14 @@ function jeAddCommands() {
   jeAddFaceCommand('properties', '', {});
   jeAddFaceCommand('radius', ' (rounded corners)', 1);
 
+  jeAddGridCommand('x', '', 103);
+  jeAddGridCommand('y', '', 160);
+  jeAddGridCommand('minX', '', 0);
+  jeAddGridCommand('minY', '', 0);
+  jeAddGridCommand('offsetX', '', 0);
+  jeAddGridCommand('offsetY', '', 0);
+  jeAddGridCommand('rotation', '', 0);
```

Comment:
Why the `description` parameter if you don't use it?

Reply:
Fixed.